### PR TITLE
TO-DO App

### PR DESCRIPTION
**Pull request by abhijeetd700**
@anilrao316

Working of the app
https://www.loom.com/share/c65edad7ab5b4b3faec1b7b3d662cbbf

### CODE REVIEW

File: Week-9/todo-app/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
```
Comment (anilrao316) [inline Week-9/todo-app/.gitignore]: Remove this file

File: Week-9/todo-app/src/Component/InputTodo.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  const [title,setTitle] = useState('')
+
+  const handleInputChange =(e)=>{
+    setTitle(e.target.value)
+  }
+
+  const handleSubmit=(e)=>{
+    e.preventDefault()
+    addTodoItem(title)
```
Comment (anilrao316) [inline Week-9/todo-app/src/Component/InputTodo.js]: Can rename the component to AddTodo even file

File: Week-9/todo-app/src/Component/TodoContainer.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  }
+    
+  return (
+    <div className='container'>
+      <div className='inner'>
+        <h1 style={{
+            fontSize: "6rem",
+            fontWeight: "600",
+            marginBottom: "2rem",
+            color: 'black',
```
Comment (anilrao316) [inline Week-9/todo-app/src/Component/TodoContainer.js]: Move this to external css

Comment (abhijeetd700) [reply Week-9/todo-app/src/Component/TodoContainer.js]: @anilrao316 
Done all suggested changes...

**Pull request merged**